cmd/webhooks/main.go

```diff
@@ -18,14 +18,16 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/MainfluxLabs/mainflux/pkg/messaging/brokers"
+	webhookshttpapi "github.com/MainfluxLabs/mainflux/webhooks/api/http"
```

Comment:
webhooks prefix is redundant, it's the webhooks service `httpapi` would be enough